src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -142,6 +142,16 @@ public VerboseProcess(final Process prc, final Level stdout,
         if (stderr == null) {
             throw new IllegalArgumentException("stderr LEVEL can't be NULL");
         }
+        if (Level.ALL.equals(stdout)) {
+            throw new IllegalArgumentException(
+                "stdout LEVEL can't be set to ALL"
+            );
+        }
+        if (Level.ALL.equals(stderr)) {
+            throw new IllegalArgumentException(
+                "stderr LEVEL can't be set to ALL"
```

Comment:
@prondzyn that's partially duplicated string, can we extract common part to a variable?